src/DynamicBanner.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FunctionComponent } from 'react';
+import { Alert } from 'antd';
+
+import { bannerActivity, bannerMessage, bannerUrl } from 'common/injectGlobals';
+
+import style from './DynamicBanner.module.scss';
+
+const SHOW_DYNAMIC_INFO_BANNER = 'SHOW_DYNAMIC_INFO_BANNER';
+
+export const hideDynamicInfoBannerForDisplay = () =>
+  localStorage.setItem(SHOW_DYNAMIC_INFO_BANNER, 'false');
+
+export const showDynamicInfoBanner = () =>
+  bannerActivity && localStorage.getItem(SHOW_DYNAMIC_INFO_BANNER) !== 'false';
```

Comment:
if you use feature toggle, you don't need bannerActivity and use `isFeatureEnabled('showDynamicBanner')` instead